### PR TITLE
Feat/pictograms

### PR DESCRIPTION
**Pull request by PeKne**
## Description
- pictograms added
## Related Issue

Related to #7997 

## Screenshots:

<details><summary>VIEW SCREENSHOTS</summary>
<p>

Snímek obrazovky 2023-04-26 v 14 56 54
Snímek obrazovky 2023-04-26 v 14 56 34
Snímek obrazovky 2023-04-26 v 14 56 09
Snímek obrazovky 2023-04-26 v 14 54 01
Snímek obrazovky 2023-04-26 v 14 53 50
Snímek obrazovky 2023-04-26 v 12 17 40

</p>
</details>

### CODE REVIEW

File: suite-native/atoms/src/Pictogram.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import { IconName, Icon } from '@trezor/icons';
+import { Color } from '@trezor/theme';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from './Box';
+import { Text } from './Text';
+import { VStack } from './Stack';
+
+export type PictogramVariant = 'green' | 'red' | 'yellow';
+type PictogramStyle = {
```
Comment (Nodonisko) [inline suite-native/atoms/src/Pictogram.tsx]: nit: place this type before PictogramVariantsMap

File: suite-native/atoms/src/Pictogram.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    icon: IconName;
+    title: string | ReactNode;
+    subtitle?: string;
+};
+
+const OUTER_RING_SIZE = 114;
+const INNER_RING_SIZE = 80;
+const ICON_SIZE = 48;
+
+const PictogramVariantsMap = {
```
Comment (Nodonisko) [inline suite-native/atoms/src/Pictogram.tsx]: nit: no need for capital

**Review by Nodonisko — APPROVED** [review]

File: suite-native/atoms/src/Pictogram.tsx (hunk 1 above)
Comment (PeKne) [reply suite-native/atoms/src/Pictogram.tsx]: fixed in 82f2db34d

File: suite-native/atoms/src/Pictogram.tsx (hunk 2 above)
Comment (PeKne) [reply suite-native/atoms/src/Pictogram.tsx]: fixed in 82f2db34d

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**